batch/README.md

```
@@ -0,0 +1,141 @@
+
+Callback jobs automatically have custom `_bid` and `_cb` attributes set with their associated Batch ID and callback type.
+
+```json
+{"jobtype":"MySuccessCallback","args":[123],"custom":{"_bid":"abc123","_cb":"success"}}
+```
+
+## Status
+
+You can retrieve the status of a batch using the BATCH STATUS command. This returns a JSON blob of data associated with the batch:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)